velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -208,6 +208,7 @@ class DecimalAggregate : public exec::Aggregate {
       });
     } else {
       rows.applyToSelected([&](vector_size_t i) {
+        clearNull(groups[i]);
```

Comment:
Is this a separate bug fix?

Reply:
It's from #4154 , I can revert these and merge that PR separately after this if you prefer.

Reply:
That would be helpful. Thanks.